client/app/account/verify/verify.html

```diff
@@ -0,0 +1,19 @@
+<navbar></navbar>
+<div class='container student'>
+  <div class='row'>
+    <div class='col-sm-12 student-courses'>
+      <h2 ng-show="verified" class="page-header">Account Verified</h2>
+      <h2 ng-hide="verified" class="page-header">Account Not Yet Verified</h2>
+      <div class="col-sm-12">
+        <div class="row">
+          <div ng-hide="verified">
+            Please check your email for a verification link.
```

Comment:
There should be a link to re-send the email, as well as some sort on instructions around it... Maybe on the signup page explain what we are using their email for. Then on this page, say you should receive an email in x minutes from x@y.com, otherwise check your spam filter or re-send it here, or change it here